pallets/subtensor/src/lib.rs

```diff
@@ -1264,6 +1269,10 @@ pub mod pallet {
         (H256, u64),
         OptionQuery,
     >;
+    #[pallet::storage]
+    /// ITEM( testnet_total_supply_override )
+    pub type TestnetTotalSupplyOverride<T> =
```

Comment:
nit: I would just call it `TotalSupplyOverride<T>` since it in theory could be used for anyway network. In practice we will only use for testnet but that is not hard-coded in any way and couldn't be really.